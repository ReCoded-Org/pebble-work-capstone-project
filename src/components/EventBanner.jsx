import Image from "next/image";
import * as React from "react";

function EventBanner({
    title,
    imageSrc,
    city,
    neighborhood,
    address,
    attendees,
    host,
}) {
    let attendeeAvatars = [];
    for (let i = 0; i < 3; i++) {
        attendeeAvatars.push(
            <svg
                key={i+1}
                className='-ml-6'
                width='45'
                height='42'
                viewBox='0 0 45 42'
                fill='none'
                xmlns='http://www.w3.org/2000/svg'
            >
                <rect width='44.8' height='42' rx='21' fill='#1A1A1A' />
                <path
                    d='M18.8836 26C18.7836 26 18.6986 25.965 18.6286 25.895C18.5586 25.825 18.5236 25.74 18.5236 25.64V15.875C18.5236 15.765 18.5586 15.675 18.6286 15.605C18.6986 15.535 18.7836 15.5 18.8836 15.5H22.7686C23.9786 15.5 24.9336 15.78 25.6336 16.34C26.3336 16.9 26.6836 17.715 26.6836 18.785C26.6836 19.525 26.4986 20.14 26.1286 20.63C25.7686 21.12 25.2836 21.475 24.6736 21.695L26.8636 25.52C26.8936 25.58 26.9086 25.635 26.9086 25.685C26.9086 25.775 26.8736 25.85 26.8036 25.91C26.7436 25.97 26.6736 26 26.5936 26H25.2286C25.0586 26 24.9286 25.955 24.8386 25.865C24.7486 25.775 24.6786 25.685 24.6286 25.595L22.6936 22.04H20.6086V25.64C20.6086 25.74 20.5736 25.825 20.5036 25.895C20.4436 25.965 20.3586 26 20.2486 26H18.8836ZM20.6086 20.345H22.7236C23.3336 20.345 23.7886 20.21 24.0886 19.94C24.3886 19.66 24.5386 19.27 24.5386 18.77C24.5386 18.27 24.3886 17.88 24.0886 17.6C23.7986 17.31 23.3436 17.165 22.7236 17.165H20.6086V20.345Z'
                    fill='white'
                />
            </svg>
        );
    }
    return (
        <div className='grid-col-1 grid grid-flow-row gap-2 md:grid-cols-3 md:gap-10 md:pl-6'>
            <h1 className='order-1 col-span-1 p-6 text-center text-2xl md:order-first md:col-span-3 md:text-left md:text-4xl'>
                {title}
            </h1>
            <div className='border-first col-span-1 flex justify-center pb-6 md:order-1 md:col-span-2 md:pl-6'>
                <Image
                    className='rounded'
                    src={imageSrc}
                    alt={`${title},image`}
                    width={800}
                    height={600}
                />
            </div>
            <div className='order-last col-span-1 flex flex-col justify-items-start justify-self-center p-6 md:items-start md:pt-48'>
                <div className='flex'>
                    <svg
                        width='42'
                        height='42'
                        viewBox='0 0 42 42'
                        fill='none'
                        xmlns='http://www.w3.org/2000/svg'
                    >
                        <path
                            d='M21.0001 3.5C13.2809 3.5 7.00014 9.78075 7.00014 17.4913C6.94939 28.77 20.4681 38.122 21.0001 38.5C21.0001 38.5 35.0509 28.77 35.0001 17.5C35.0001 9.78075 28.7194 3.5 21.0001 3.5ZM21.0001 24.5C17.1326 24.5 14.0001 21.3675 14.0001 17.5C14.0001 13.6325 17.1326 10.5 21.0001 10.5C24.8676 10.5 28.0001 13.6325 28.0001 17.5C28.0001 21.3675 24.8676 24.5 21.0001 24.5Z'
                            fill='black'
                        />
                    </svg>
                    <div>
                        <p>
                            <b>{city}</b>
                        </p>
                        <p>{neighborhood}</p>
                        <p>{address}</p>
                    </div>
                </div>
                <div className='flex items-center pt-6'>
                    {attendeeAvatars}
                    <p className='pl-6'>{`+${attendees.length} Attendees`}</p>
                </div>
                <div className='flex items-center pt-6 pb-12'>
                    <svg
                        className='-ml-6'
                        width='45'
                        height='42'
                        viewBox='0 0 45 42'
                        fill='none'
                        xmlns='http://www.w3.org/2000/svg'
                    >
                        <rect width='44.8' height='42' rx='21' fill='#1A1A1A' />
                        <path
                            d='M18.8836 26C18.7836 26 18.6986 25.965 18.6286 25.895C18.5586 25.825 18.5236 25.74 18.5236 25.64V15.875C18.5236 15.765 18.5586 15.675 18.6286 15.605C18.6986 15.535 18.7836 15.5 18.8836 15.5H22.7686C23.9786 15.5 24.9336 15.78 25.6336 16.34C26.3336 16.9 26.6836 17.715 26.6836 18.785C26.6836 19.525 26.4986 20.14 26.1286 20.63C25.7686 21.12 25.2836 21.475 24.6736 21.695L26.8636 25.52C26.8936 25.58 26.9086 25.635 26.9086 25.685C26.9086 25.775 26.8736 25.85 26.8036 25.91C26.7436 25.97 26.6736 26 26.5936 26H25.2286C25.0586 26 24.9286 25.955 24.8386 25.865C24.7486 25.775 24.6786 25.685 24.6286 25.595L22.6936 22.04H20.6086V25.64C20.6086 25.74 20.5736 25.825 20.5036 25.895C20.4436 25.965 20.3586 26 20.2486 26H18.8836ZM20.6086 20.345H22.7236C23.3336 20.345 23.7886 20.21 24.0886 19.94C24.3886 19.66 24.5386 19.27 24.5386 18.77C24.5386 18.27 24.3886 17.88 24.0886 17.6C23.7986 17.31 23.3436 17.165 22.7236 17.165H20.6086V20.345Z'
                            fill='white'
                        />
                    </svg>
                    <p className='pl-6'>{`Organized by ${host}`}</p>
                </div>
                <button className='h-12 w-48 rounded bg-primary-200'>
                    Join
                </button>
            </div>
        </div>
    );
}

export default EventBanner;
