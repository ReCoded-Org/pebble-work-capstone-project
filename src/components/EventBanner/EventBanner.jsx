import Image from "next/image";
import * as React from "react";
import { useState } from "react";

import Button from "@/components/Button";

function EventBanner({
    title,
    imageSrc,
    city,
    neighborhood,
    address,
    attendees,
    host,
}) {
    const [join, setJoin] = useState("Join");
    function handleJoinClick() {
        if (join === "Join") {
            setJoin("Leave");
        } else {
            setJoin("Join");
        }
    }
    let attendeeAvatars = [];
    attendeeAvatars.push(
        <div className=''>
            <Image
                className='rounded-full'
                src='/images/userAvatar.jpeg'
                alt={`${title},image`}
                width={48}
                height={48}
            />
        </div>
    );
    for (let i = 0; i < 3; i++) {
        attendeeAvatars.push(
            <div className='-ml-6'>
                <Image
                    className='rounded-full'
                    src='/images/userAvatar.jpeg'
                    alt={`${title},image`}
                    width={48}
                    height={48}
                />
            </div>
        );
    }
    return (
        <div className='grid-col-1 grid grid-flow-row gap-2 md:grid-cols-3 md:gap-2 md:p-6'>
            <h1 className='order-1 col-span-1 p-6 text-center text-2xl font-semibold md:order-first md:col-span-3 md:text-left md:text-4xl md:font-normal'>
                {title}
            </h1>
            <div className='border-first col-span-1 flex justify-center px-6 md:order-1 md:col-span-2 md:pl-0 md:pb-6'>
                <Image
                    className='rounded'
                    src={imageSrc}
                    alt={`${title},image`}
                    width={600}
                    height={400}
                />
            </div>
            <div className='order-last col-span-1 flex flex-col items-center pb-6  md:items-start md:justify-items-start md:pt-12'>
                <div className='flex'>
                    <svg
                        width='42'
                        height='42'
                        viewBox='0 0 42 42'
                        fill='none'
                        xmlns='http://www.w3.org/2000/svg'
                    >
                        <path
                            d='M21.0001 3.5C13.2809 3.5 7.00014 9.78075 7.00014 17.4913C6.94939 28.77 20.4681 38.122 21.0001 38.5C21.0001 38.5 35.0509 28.77 35.0001 17.5C35.0001 9.78075 28.7194 3.5 21.0001 3.5ZM21.0001 24.5C17.1326 24.5 14.0001 21.3675 14.0001 17.5C14.0001 13.6325 17.1326 10.5 21.0001 10.5C24.8676 10.5 28.0001 13.6325 28.0001 17.5C28.0001 21.3675 24.8676 24.5 21.0001 24.5Z'
                            fill='black'
                        />
                    </svg>
                    <div>
                        <p>
                            <b>{city}</b>
                        </p>
                        <p>{neighborhood}</p>
                        <p>{address}</p>
                    </div>
                </div>
                <div className='flex items-center pt-6'>
                    {attendeeAvatars}
                    <p className='pl-6'>{`+${attendees.length} Attendees`}</p>
                </div>
                <div className='flex items-center pt-6 pb-12'>
                    <Image
                        className='rounded-full'
                        src='/images/userAvatar.jpeg'
                        alt='avatar image'
                        width={48}
                        height={48}
                    />
                    <p className='pl-6'>{`Organized by ${host}`}</p>
                </div>
                <Button
                    label={join}
                    bgColor='bg-primary-200'
                    textColor='text-white'
                    height='h-12'
                    width='w-48'
                    onClick={handleJoinClick}
                />
            </div>
        </div>
    );
}

export default EventBanner;
