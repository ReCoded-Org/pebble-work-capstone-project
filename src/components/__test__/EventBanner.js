import renderer from "react-test-renderer";

import EventBanner from "../EventBanner";

it("renders correctly", () => {
    const tree = renderer.create(<EventBanner />).toJSON();
    expect(tree).toMatchSnapshot();
});
