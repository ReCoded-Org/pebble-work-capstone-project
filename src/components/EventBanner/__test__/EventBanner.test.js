import renderer from "react-test-renderer";

import EventBanner from "../EventBanner";

it("renders correctly", () => {
    const tree = renderer
        .create(
            <EventBanner
                title='This is some title This is some title This is some title'
                imageSrc='/images/sampleEventImage.jpg'
                city='Izmir'
                neighborhood='Torbaci'
                address='11th Street South, Arlington, Virginia'
                attendees={[
                    "steve",
                    "jane",
                    "douglas",
                    "whatever",
                    "beyonce",
                    "bomm",
                ]}
                host='Jangis Khan'
            />
        )
        .toJSON();
    expect(tree).toMatchSnapshot();
});
