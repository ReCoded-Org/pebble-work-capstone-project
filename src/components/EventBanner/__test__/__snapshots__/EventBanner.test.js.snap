// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="grid-col-1 grid grid-flow-row gap-2 md:grid-cols-3 md:gap-2 md:pl-6"
>
  <h1
    className="order-1 col-span-1 p-6 text-center text-2xl md:order-first md:col-span-3 md:text-left md:text-4xl"
  >
    Izmir Clean Energy Brainstorming and Slashing Things Up Meetup (with a very long title)
  </h1>
  <div
    className="border-first col-span-1 flex justify-center pb-6 md:order-1 md:col-span-2 md:pl-6"
  >
    <span
      style={
        Object {
          "background": "none",
          "border": 0,
          "boxSizing": "border-box",
          "display": "inline-block",
          "height": "initial",
          "margin": 0,
          "maxWidth": "100%",
          "opacity": 1,
          "overflow": "hidden",
          "padding": 0,
          "position": "relative",
          "width": "initial",
        }
      }
    >
      <span
        style={
          Object {
            "background": "none",
            "border": 0,
            "boxSizing": "border-box",
            "display": "block",
            "height": "initial",
            "margin": 0,
            "maxWidth": "100%",
            "opacity": 1,
            "padding": 0,
            "width": "initial",
          }
        }
      >
        <img
          alt=""
          aria-hidden={true}
          src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%27600%27%20height=%27400%27/%3e"
          style={
            Object {
              "background": "none",
              "border": 0,
              "display": "block",
              "height": "initial",
              "margin": 0,
              "maxWidth": "100%",
              "opacity": 1,
              "padding": 0,
              "width": "initial",
            }
          }
        />
      </span>
      <img
        alt="Izmir Clean Energy Brainstorming and Slashing Things Up Meetup (with a very long title),image"
        className="rounded"
        data-nimg="intrinsic"
        decoding="async"
        onError={[Function]}
        onLoad={[Function]}
        src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
        style={
          Object {
            "border": "none",
            "bottom": 0,
            "boxSizing": "border-box",
            "display": "block",
            "height": 0,
            "left": 0,
            "margin": "auto",
            "maxHeight": "100%",
            "maxWidth": "100%",
            "minHeight": "100%",
            "minWidth": "100%",
            "objectFit": undefined,
            "objectPosition": undefined,
            "padding": 0,
            "position": "absolute",
            "right": 0,
            "top": 0,
            "width": 0,
          }
        }
      />
      <noscript>
        <img
          alt="Izmir Clean Energy Brainstorming and Slashing Things Up Meetup (with a very long title),image"
          className="rounded"
          data-nimg="intrinsic"
          decoding="async"
          loading="lazy"
          src="/_next/image?url=%2Fimages%2FsampleEventImage.jpg&w=1200&q=75"
          srcSet="/_next/image?url=%2Fimages%2FsampleEventImage.jpg&w=640&q=75 1x, /_next/image?url=%2Fimages%2FsampleEventImage.jpg&w=1200&q=75 2x"
          style={
            Object {
              "border": "none",
              "bottom": 0,
              "boxSizing": "border-box",
              "display": "block",
              "height": 0,
              "left": 0,
              "margin": "auto",
              "maxHeight": "100%",
              "maxWidth": "100%",
              "minHeight": "100%",
              "minWidth": "100%",
              "objectFit": undefined,
              "objectPosition": undefined,
              "padding": 0,
              "position": "absolute",
              "right": 0,
              "top": 0,
              "width": 0,
            }
          }
        />
      </noscript>
    </span>
  </div>
  <div
    className="order-last col-span-1 flex flex-col items-center pb-6 md:items-start md:justify-items-start md:pt-12"
  >
    <div
      className="flex"
    >
      <svg
        fill="none"
        height="42"
        viewBox="0 0 42 42"
        width="42"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M21.0001 3.5C13.2809 3.5 7.00014 9.78075 7.00014 17.4913C6.94939 28.77 20.4681 38.122 21.0001 38.5C21.0001 38.5 35.0509 28.77 35.0001 17.5C35.0001 9.78075 28.7194 3.5 21.0001 3.5ZM21.0001 24.5C17.1326 24.5 14.0001 21.3675 14.0001 17.5C14.0001 13.6325 17.1326 10.5 21.0001 10.5C24.8676 10.5 28.0001 13.6325 28.0001 17.5C28.0001 21.3675 24.8676 24.5 21.0001 24.5Z"
          fill="black"
        />
      </svg>
      <div>
        <p>
          <b>
            Izmir, Turkey
          </b>
        </p>
        <p>
          Mimar Sinan Mahallesi
        </p>
        <p>
          26 Ağustos Kapısı
        </p>
      </div>
    </div>
    <div
      className="flex items-center pt-6"
    >
      <div
        className=""
      >
        <span
          style={
            Object {
              "background": "none",
              "border": 0,
              "boxSizing": "border-box",
              "display": "inline-block",
              "height": "initial",
              "margin": 0,
              "maxWidth": "100%",
              "opacity": 1,
              "overflow": "hidden",
              "padding": 0,
              "position": "relative",
              "width": "initial",
            }
          }
        >
          <span
            style={
              Object {
                "background": "none",
                "border": 0,
                "boxSizing": "border-box",
                "display": "block",
                "height": "initial",
                "margin": 0,
                "maxWidth": "100%",
                "opacity": 1,
                "padding": 0,
                "width": "initial",
              }
            }
          >
            <img
              alt=""
              aria-hidden={true}
              src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2748%27%20height=%2748%27/%3e"
              style={
                Object {
                  "background": "none",
                  "border": 0,
                  "display": "block",
                  "height": "initial",
                  "margin": 0,
                  "maxWidth": "100%",
                  "opacity": 1,
                  "padding": 0,
                  "width": "initial",
                }
              }
            />
          </span>
          <img
            alt="Izmir Clean Energy Brainstorming and Slashing Things Up Meetup (with a very long title),image"
            className="rounded-full"
            data-nimg="intrinsic"
            decoding="async"
            onError={[Function]}
            onLoad={[Function]}
            src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
            style={
              Object {
                "border": "none",
                "bottom": 0,
                "boxSizing": "border-box",
                "display": "block",
                "height": 0,
                "left": 0,
                "margin": "auto",
                "maxHeight": "100%",
                "maxWidth": "100%",
                "minHeight": "100%",
                "minWidth": "100%",
                "objectFit": undefined,
                "objectPosition": undefined,
                "padding": 0,
                "position": "absolute",
                "right": 0,
                "top": 0,
                "width": 0,
              }
            }
          />
          <noscript>
            <img
              alt="Izmir Clean Energy Brainstorming and Slashing Things Up Meetup (with a very long title),image"
              className="rounded-full"
              data-nimg="intrinsic"
              decoding="async"
              loading="lazy"
              src="/_next/image?url=%2Fimages%2FuserAvatar.jpeg&w=96&q=75"
              srcSet="/_next/image?url=%2Fimages%2FuserAvatar.jpeg&w=48&q=75 1x, /_next/image?url=%2Fimages%2FuserAvatar.jpeg&w=96&q=75 2x"
              style={
                Object {
                  "border": "none",
                  "bottom": 0,
                  "boxSizing": "border-box",
                  "display": "block",
                  "height": 0,
                  "left": 0,
                  "margin": "auto",
                  "maxHeight": "100%",
                  "maxWidth": "100%",
                  "minHeight": "100%",
                  "minWidth": "100%",
                  "objectFit": undefined,
                  "objectPosition": undefined,
                  "padding": 0,
                  "position": "absolute",
                  "right": 0,
                  "top": 0,
                  "width": 0,
                }
              }
            />
          </noscript>
        </span>
      </div>
      <div
        className="-ml-6"
      >
        <span
          style={
            Object {
              "background": "none",
              "border": 0,
              "boxSizing": "border-box",
              "display": "inline-block",
              "height": "initial",
              "margin": 0,
              "maxWidth": "100%",
              "opacity": 1,
              "overflow": "hidden",
              "padding": 0,
              "position": "relative",
              "width": "initial",
            }
          }
        >
          <span
            style={
              Object {
                "background": "none",
                "border": 0,
                "boxSizing": "border-box",
                "display": "block",
                "height": "initial",
                "margin": 0,
                "maxWidth": "100%",
                "opacity": 1,
                "padding": 0,
                "width": "initial",
              }
            }
          >
            <img
              alt=""
              aria-hidden={true}
              src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2748%27%20height=%2748%27/%3e"
              style={
                Object {
                  "background": "none",
                  "border": 0,
                  "display": "block",
                  "height": "initial",
                  "margin": 0,
                  "maxWidth": "100%",
                  "opacity": 1,
                  "padding": 0,
                  "width": "initial",
                }
              }
            />
          </span>
          <img
            alt="Izmir Clean Energy Brainstorming and Slashing Things Up Meetup (with a very long title),image"
            className="rounded-full"
            data-nimg="intrinsic"
            decoding="async"
            onError={[Function]}
            onLoad={[Function]}
            src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
            style={
              Object {
                "border": "none",
                "bottom": 0,
                "boxSizing": "border-box",
                "display": "block",
                "height": 0,
                "left": 0,
                "margin": "auto",
                "maxHeight": "100%",
                "maxWidth": "100%",
                "minHeight": "100%",
                "minWidth": "100%",
                "objectFit": undefined,
                "objectPosition": undefined,
                "padding": 0,
                "position": "absolute",
                "right": 0,
                "top": 0,
                "width": 0,
              }
            }
          />
          <noscript>
            <img
              alt="Izmir Clean Energy Brainstorming and Slashing Things Up Meetup (with a very long title),image"
              className="rounded-full"
              data-nimg="intrinsic"
              decoding="async"
              loading="lazy"
              src="/_next/image?url=%2Fimages%2FuserAvatar.jpeg&w=96&q=75"
              srcSet="/_next/image?url=%2Fimages%2FuserAvatar.jpeg&w=48&q=75 1x, /_next/image?url=%2Fimages%2FuserAvatar.jpeg&w=96&q=75 2x"
              style={
                Object {
                  "border": "none",
                  "bottom": 0,
                  "boxSizing": "border-box",
                  "display": "block",
                  "height": 0,
                  "left": 0,
                  "margin": "auto",
                  "maxHeight": "100%",
                  "maxWidth": "100%",
                  "minHeight": "100%",
                  "minWidth": "100%",
                  "objectFit": undefined,
                  "objectPosition": undefined,
                  "padding": 0,
                  "position": "absolute",
                  "right": 0,
                  "top": 0,
                  "width": 0,
                }
              }
            />
          </noscript>
        </span>
      </div>
      <div
        className="-ml-6"
      >
        <span
          style={
            Object {
              "background": "none",
              "border": 0,
              "boxSizing": "border-box",
              "display": "inline-block",
              "height": "initial",
              "margin": 0,
              "maxWidth": "100%",
              "opacity": 1,
              "overflow": "hidden",
              "padding": 0,
              "position": "relative",
              "width": "initial",
            }
          }
        >
          <span
            style={
              Object {
                "background": "none",
                "border": 0,
                "boxSizing": "border-box",
                "display": "block",
                "height": "initial",
                "margin": 0,
                "maxWidth": "100%",
                "opacity": 1,
                "padding": 0,
                "width": "initial",
              }
            }
          >
            <img
              alt=""
              aria-hidden={true}
              src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2748%27%20height=%2748%27/%3e"
              style={
                Object {
                  "background": "none",
                  "border": 0,
                  "display": "block",
                  "height": "initial",
                  "margin": 0,
                  "maxWidth": "100%",
                  "opacity": 1,
                  "padding": 0,
                  "width": "initial",
                }
              }
            />
          </span>
          <img
            alt="Izmir Clean Energy Brainstorming and Slashing Things Up Meetup (with a very long title),image"
            className="rounded-full"
            data-nimg="intrinsic"
            decoding="async"
            onError={[Function]}
            onLoad={[Function]}
            src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
            style={
              Object {
                "border": "none",
                "bottom": 0,
                "boxSizing": "border-box",
                "display": "block",
                "height": 0,
                "left": 0,
                "margin": "auto",
                "maxHeight": "100%",
                "maxWidth": "100%",
                "minHeight": "100%",
                "minWidth": "100%",
                "objectFit": undefined,
                "objectPosition": undefined,
                "padding": 0,
                "position": "absolute",
                "right": 0,
                "top": 0,
                "width": 0,
              }
            }
          />
          <noscript>
            <img
              alt="Izmir Clean Energy Brainstorming and Slashing Things Up Meetup (with a very long title),image"
              className="rounded-full"
              data-nimg="intrinsic"
              decoding="async"
              loading="lazy"
              src="/_next/image?url=%2Fimages%2FuserAvatar.jpeg&w=96&q=75"
              srcSet="/_next/image?url=%2Fimages%2FuserAvatar.jpeg&w=48&q=75 1x, /_next/image?url=%2Fimages%2FuserAvatar.jpeg&w=96&q=75 2x"
              style={
                Object {
                  "border": "none",
                  "bottom": 0,
                  "boxSizing": "border-box",
                  "display": "block",
                  "height": 0,
                  "left": 0,
                  "margin": "auto",
                  "maxHeight": "100%",
                  "maxWidth": "100%",
                  "minHeight": "100%",
                  "minWidth": "100%",
                  "objectFit": undefined,
                  "objectPosition": undefined,
                  "padding": 0,
                  "position": "absolute",
                  "right": 0,
                  "top": 0,
                  "width": 0,
                }
              }
            />
          </noscript>
        </span>
      </div>
      <div
        className="-ml-6"
      >
        <span
          style={
            Object {
              "background": "none",
              "border": 0,
              "boxSizing": "border-box",
              "display": "inline-block",
              "height": "initial",
              "margin": 0,
              "maxWidth": "100%",
              "opacity": 1,
              "overflow": "hidden",
              "padding": 0,
              "position": "relative",
              "width": "initial",
            }
          }
        >
          <span
            style={
              Object {
                "background": "none",
                "border": 0,
                "boxSizing": "border-box",
                "display": "block",
                "height": "initial",
                "margin": 0,
                "maxWidth": "100%",
                "opacity": 1,
                "padding": 0,
                "width": "initial",
              }
            }
          >
            <img
              alt=""
              aria-hidden={true}
              src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2748%27%20height=%2748%27/%3e"
              style={
                Object {
                  "background": "none",
                  "border": 0,
                  "display": "block",
                  "height": "initial",
                  "margin": 0,
                  "maxWidth": "100%",
                  "opacity": 1,
                  "padding": 0,
                  "width": "initial",
                }
              }
            />
          </span>
          <img
            alt="Izmir Clean Energy Brainstorming and Slashing Things Up Meetup (with a very long title),image"
            className="rounded-full"
            data-nimg="intrinsic"
            decoding="async"
            onError={[Function]}
            onLoad={[Function]}
            src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
            style={
              Object {
                "border": "none",
                "bottom": 0,
                "boxSizing": "border-box",
                "display": "block",
                "height": 0,
                "left": 0,
                "margin": "auto",
                "maxHeight": "100%",
                "maxWidth": "100%",
                "minHeight": "100%",
                "minWidth": "100%",
                "objectFit": undefined,
                "objectPosition": undefined,
                "padding": 0,
                "position": "absolute",
                "right": 0,
                "top": 0,
                "width": 0,
              }
            }
          />
          <noscript>
            <img
              alt="Izmir Clean Energy Brainstorming and Slashing Things Up Meetup (with a very long title),image"
              className="rounded-full"
              data-nimg="intrinsic"
              decoding="async"
              loading="lazy"
              src="/_next/image?url=%2Fimages%2FuserAvatar.jpeg&w=96&q=75"
              srcSet="/_next/image?url=%2Fimages%2FuserAvatar.jpeg&w=48&q=75 1x, /_next/image?url=%2Fimages%2FuserAvatar.jpeg&w=96&q=75 2x"
              style={
                Object {
                  "border": "none",
                  "bottom": 0,
                  "boxSizing": "border-box",
                  "display": "block",
                  "height": 0,
                  "left": 0,
                  "margin": "auto",
                  "maxHeight": "100%",
                  "maxWidth": "100%",
                  "minHeight": "100%",
                  "minWidth": "100%",
                  "objectFit": undefined,
                  "objectPosition": undefined,
                  "padding": 0,
                  "position": "absolute",
                  "right": 0,
                  "top": 0,
                  "width": 0,
                }
              }
            />
          </noscript>
        </span>
      </div>
      <p
        className="pl-6"
      >
        +4 Attendees
      </p>
    </div>
    <div
      className="flex items-center pt-6 pb-12"
    >
      <span
        style={
          Object {
            "background": "none",
            "border": 0,
            "boxSizing": "border-box",
            "display": "inline-block",
            "height": "initial",
            "margin": 0,
            "maxWidth": "100%",
            "opacity": 1,
            "overflow": "hidden",
            "padding": 0,
            "position": "relative",
            "width": "initial",
          }
        }
      >
        <span
          style={
            Object {
              "background": "none",
              "border": 0,
              "boxSizing": "border-box",
              "display": "block",
              "height": "initial",
              "margin": 0,
              "maxWidth": "100%",
              "opacity": 1,
              "padding": 0,
              "width": "initial",
            }
          }
        >
          <img
            alt=""
            aria-hidden={true}
            src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2748%27%20height=%2748%27/%3e"
            style={
              Object {
                "background": "none",
                "border": 0,
                "display": "block",
                "height": "initial",
                "margin": 0,
                "maxWidth": "100%",
                "opacity": 1,
                "padding": 0,
                "width": "initial",
              }
            }
          />
        </span>
        <img
          alt="avatar image"
          className="rounded-full"
          data-nimg="intrinsic"
          decoding="async"
          onError={[Function]}
          onLoad={[Function]}
          src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
          style={
            Object {
              "border": "none",
              "bottom": 0,
              "boxSizing": "border-box",
              "display": "block",
              "height": 0,
              "left": 0,
              "margin": "auto",
              "maxHeight": "100%",
              "maxWidth": "100%",
              "minHeight": "100%",
              "minWidth": "100%",
              "objectFit": undefined,
              "objectPosition": undefined,
              "padding": 0,
              "position": "absolute",
              "right": 0,
              "top": 0,
              "width": 0,
            }
          }
        />
        <noscript>
          <img
            alt="avatar image"
            className="rounded-full"
            data-nimg="intrinsic"
            decoding="async"
            loading="lazy"
            src="/_next/image?url=%2Fimages%2FuserAvatar.jpeg&w=96&q=75"
            srcSet="/_next/image?url=%2Fimages%2FuserAvatar.jpeg&w=48&q=75 1x, /_next/image?url=%2Fimages%2FuserAvatar.jpeg&w=96&q=75 2x"
            style={
              Object {
                "border": "none",
                "bottom": 0,
                "boxSizing": "border-box",
                "display": "block",
                "height": 0,
                "left": 0,
                "margin": "auto",
                "maxHeight": "100%",
                "maxWidth": "100%",
                "minHeight": "100%",
                "minWidth": "100%",
                "objectFit": undefined,
                "objectPosition": undefined,
                "padding": 0,
                "position": "absolute",
                "right": 0,
                "top": 0,
                "width": 0,
              }
            }
          />
        </noscript>
      </span>
      <p
        className="pl-6"
      >
        Organized by Jangis M.
      </p>
    </div>
    <button
      className="m-2 w-48 h-12 bg-primary-200  hover:undefined text-white undefined  py-1  px-2 undefined hover:undefined rounded  hover:shadow-xl"
      onClick={[Function]}
      type="button"
    >
      Join
    </button>
  </div>
</div>
`;
